deploy/examples/auto-provision-kafka.yaml

```diff
@@ -0,0 +1,26 @@
+# this example requires the strimzi operator to be running in the cluster:
+# https://strimzi.io/quickstarts/minikube/
+# basically:
+# $ kubectl create namespace kafka
+# $ curl -L https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.14.0/strimzi-cluster-operator-0.14.0.yaml \
+#  | sed 's/namespace: .*/namespace: kafka/' \
+#  | kubectl apply -f - -n kafka 
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: auto-provision-kafka
+spec:
+  strategy: streaming
+  collector:
+    image: jaegertracing/jaeger-collector:latest # TLS configuration is supported starting from 1.15
```

Comment:
Is this and the ingester image temporary as you need the latest image?

Reply:
Correct